### DISCUSSION TITLE
How do I connect a device?

### DISCUSSION THREAD

**Discussion by mukeenanyafiq** [Paving-Base/APK-Installer#189] [Q&A] [answered]
Hi, I want to ask a question
How do I connect a device in APK Installer?
Do another device must have something and do something?

**Comment by wherewhere** [comment, marked as answer]
You can go to setting page to choose other device. Remember turn Is Only WSA off.

**Reply by mukeenanyafiq** [reply]
I know, but, in another device, how am I supposed to do to connect it into APK Installer?

**Reply by wherewhere** [reply]
> I know, but, in another device, how am I supposed to do to connect it into APK Installer?

In Setting Page.

**Reply by mukeenanyafiq** [reply]
Nevermind. I found this in your apps rating page. Thank you.
![image](https://user-images.githubusercontent.com/93072673/168773570-4045d85d-73b2-407e-8ff4-8268255da490.png)

**Reply by wherewhere** [reply]
LOL.

**Reply by mukeenanyafiq** [reply]
bruh you didnt understand

**Reply by wherewhere** [reply]
???

**Reply by mukeenanyafiq** [reply]
nevermind tho.
